beats/metricbeat/install.sls

```diff
@@ -5,5 +5,6 @@ include:
 metricbeat_install:
     pkg.installed:
         - name: metricbeat
+        - version: {{ salt['pillar.get']('beats:metricbeat:version', '6.1.2') }}
```

Comment:
Why not something like that?

```yaml
[...]
{% if beats.metricbeat.version is defined %}
- version: {{ salt['pillar.get']('beats:metricbeat:version') }}
{% endif %}
[...]
```
If specified within the pillar, the version will be enforced. If not, the latest version of the package will be installed